### ISSUE TITLE
How to expand and list members of a DL

### ISSUE THREAD

**Issue by Gugla** [O365/python-o365#613]
I have a requirement where a DL needs to be list to read its members and perform the subsequent operations based on whether a particular member is a part of that DL or not. I can't find any property/function to achieve this functionality in O365.

I thought of using the query helper but not sure what would be the 'attribute' in this case. I was earlier using EWS with the below query to achieve this.

**EWS**

```
for ids in account.protocol.expand_dl(sender):
     print(ids.email_address)
```

Can we do a similar type of query on the distribution list in O365 to list the members?

**Comment by janscas** [comment]
I don't think so. Last time I check that on the MS Graph docs I couldn't find anything.

If you find something let me know.
